src/config/configurable.rs

```diff
@@ -171,6 +177,17 @@ impl ConfigurableDb for TabletRegistry<RocksEngine> {
         })
     }
 
+    fn set_cf_flush_size(&self, cf: &str, f: usize) -> ConfigRes {
+        loop_registry(self, |cache| {
```

Comment:
Do you need to limit the size range?

Reply:
It's not limited before for global write buffer manager.